docs/user_guide/models/peakvi.rst

```diff
@@ -0,0 +1,170 @@
+
+The latent variables, along with their description are summarized in the following table:
+
+.. list-table::
+   :widths: 20 90 15
+   :header-rows: 1
+
+   * - Latent variable
+     - Description
+     - Code variable (if different)
+   * - :math:`z_i \in \mathbb{R}^d`
+     - Low-dimensional representation capturing the state of a cell
+     - ``z``
+   * - :math:`\y_i \in \left[0,1\right]^{M}`
+     - Acessibility probability estimate
+     - ``p``
+   * - :math:`\ell_i \in \left[0,1\right]`
+     - Cell-wise scaling factor
+     - ``d``
+   * - :math:`r_j \in \left[0,1\right]`
+     - Region-wise scaling factor
+     - ``f``
+
+Inference
+=========
+
+PeakVI uses variational inference, specifically auto-encoding variational Bayes (see :doc:`/user_guide/background/variational_inference`) to learn both the model parameters (the neural network params, scaling factors, etc.) and an approximate posterior distribution with the following factorization:
+
+ .. math::
+    :nowrap:
+
+    \begin{align}
+       q_\eta(z_i, \ell_i \mid x_i) :=
+       q_\eta(z_i \mid x_i)q_\eta(\ell_i \mid x_i).
+    \end{align}
+
+Here :math:`\eta` is a set of parameters corresponding to inference neural networks (encoders), which we do not describe in detail here. The underlying class used as the encoder for PeakVI is :class:`~scvi.nn.Encoder`.
+
+It it important to note that by default, PeakVI only receives the accessibility data as input (i.e., not the observed cell-level covariates).
+Empirically, we have not seen much of a difference by having the encoder take as input the concatenation of these items (i.e., :math:`q_\eta(z_i, \ell_i \mid x_i, s_i)`, but users can control it manually by passing
+``encode_covariates=True`` to :class:`scvi.model.PEAKVI`.
+
+Tasks
+=====
+
+Here we provide an overview of some of the tasks that PeakVI can perform. Please see :class:`scvi.model.PEAKVI` for the full API reference.
+
+Dimensionality reduction
+------------------------
+For dimensionality reduction, the mean of the approximate posterior :math:`q_\eta(z_i \mid x_i)` is returned by default.
+This is achieved using the method::
+
+    >>> adata.obsm["X_peakvi"] = model.get_latent_representation()
+
+Users may also return samples from this distribution, as opposed to the mean, by passing the argument ``give_mean=False``.
+The latent representation can be used to create a nearest neighbor graph with scanpy with::
+
+    >>> import scanpy as sc
+    >>> sc.pp.neighbors(adata, use_rep="X_scvi")
+    >>> adata.obsp["distances"]
+
+
+Transfer learning
+-----------------
+
+A PeakVI model can be pre-trained on reference data and updated with query data using :func:`~scvi.model.PEAKVI.load_query_data`, which then facilitates transfer of metadata like cell type annotations. See the :doc:`/user_guide/background/transfer_learning` guide for more information.
+
+
+Estimation of accessibility
+------------------------------------------------
```

Comment:
I thought you feed the mean of z through the decoder? We should note this here

Reply:
yup, my bad. Fixed.